### PR TITLE
Create threads UI + loading

### PR DESCRIPTION
**Pull request by T-Damer**

### CODE REVIEW

File: src/components/BlockchainList/CommentBody.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  replier: string
+  repliedTo: string
+  timestamp: number
+}) {
+  const { current } = useSnapshot(TimeStore)
+  const [inputOpen, setInputOpen] = useState(false)
+  const formattedTime = formatDate(timestamp, current)
+
+  return (
+    <div className={commentWithInput} data-anchor={`#reply=1`}>
```
Comment (T-Damer) [inline src/components/BlockchainList/CommentBody.tsx:81]: Just some mock data, should be replaced by real one

File: src/components/BlockchainList/CommentWithReplies.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  display,
+  flex,
+  flexDirection,
+  gap,
+  margin,
+  transitionProperty,
+  width,
+} from 'classnames/tailwind'
+
+const repliesWithLine = classnames(display('flex'), margin('mt-3'))
```
Comment (T-Damer) [inline src/components/BlockchainList/CommentWithReplies.tsx:15]: `margin` can't be avoided here, I tried

File: src/components/BlockchainList/CommentWithReplies.tsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+const repliesBlock = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  flex('flex-1'),
+  gap('gap-y-3')
+)
+const commentLine = classnames(
+  width('w-4'),
+  borderWidth('border-l'),
```
Comment (T-Damer) [inline src/components/BlockchainList/CommentWithReplies.tsx:24]: This is `border-L` :)

File: src/components/BlockchainList/ReplyInput.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+  const [text, setText] = useState('')
+
+  return (
+    <>
+      <TextArea
+        text={text}
+        placeholder={`Reply to ${truncateMiddleIfNeeded(replyingTo, 12)}`}
+        onTextChange={setText}
+        maxLength={280}
```
Comment (T-Damer) [inline src/components/BlockchainList/ReplyInput.tsx:42]: Maybe we have a farcaster post `maxLenght` stored somewhere?

File: src/components/BlockchainList/ThreadPart.tsx [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+import Replies from 'components/BlockchainList/Replies'
+
+const wrapper = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  gap('gap-y-3')
+)
+
+export default function ({ replyingTo }: { replyingTo: string }) {
+  const comments = [
```
Comment (T-Damer) [inline src/components/BlockchainList/ThreadPart.tsx:12]: Just mock data, I hope it represents the real data, didn't find how it looks like in farcaster docs

Comment (upacyxou) [inline src/components/BlockchainList/ThreadPart.tsx:12]: do you need this? maybe add a "TODO" comment?

**Review by upacyxou — APPROVED** [review]

**Pull request merged**